### DISCUSSION TITLE
Changing robots.txt in Directus Cloud

### DISCUSSION THREAD

**Discussion by alexreid** [directus/directus#17899] [Archive]
#17448 allows robots.txt to be customised, which is a great start, but unfortunately, I don't think there's any way for paying Directus Cloud customers to take advantage of this. Due to the blanket disallow in the default robots.txt, web crawlers (Google etc) are not allowed to load the images, which causes impact to the experience score for any pages embedding images from Directus. This also affects cases where images stored in Directus are used in page metadata (such as Twitter cards and Open Graph images).

Is there a way to change the contents of the robots.txt response for Directus Cloud?

**Comment by SimonEdoki** [comment]
My entire website fails to get indexed by Goggle because of this. The website depends on content from Directus to load. Support was no help and the current proposed solution is to pay thousands of dollars for the "enterprise" version – about 20 times the price of the "professional" tier. This makes "professional" tier completely useless. I don't understand how this isn't a bigger deal...

**Comment by Tolmark12** [comment]
I'm dealing with this as well–specifically twitter is unable to load my images

**Comment by rijkvanzanten** [comment]
We've updated our infra so this can be changed on a self-service project-by-project basis. Our support team can now set it on your behalf (just shoot a quick email to support@directus.io) and it'll soon launch as a diy option in the cloud dashboard as well 🙂
